Portal/src/Datahub.Portal/Pages/Workspace/WebApp/WorkspaceWebAppPage.razor

```diff
@@ -0,0 +1,186 @@
+﻿@using Datahub.Application.Services.Security
+@using Datahub.Portal.Views.Dialogs
+@using Datahub.Shared.Entities
+@using Microsoft.Identity.Web
+@using System.Text.Json.Nodes
+@using Datahub.Infrastructure.Queues.Messages
+@using MediatR
+
+@inject IDbContextFactory<DatahubProjectDBContext> _dbContextFactory
+@inject IDialogService _dialogService
+@inject ISnackbar _snackbar
+@inject ILogger<WorkspaceWebAppPage> _logger
+@inject IMediator _mediatr
+
+<MudStack>
+    <MudStack Row="true">
+        <MudText Typo="Typo.h1">
+            @Localizer["Web Application"]
+        </MudText>
+        <MudSpacer/>
+        @if (_isProvisioned)
+        {
+            <MudButton Color="Color.Primary" Size="Size.Medium" Variant="Variant.Filled" OnClick="async () => { await ConfigureWebApp(); }">
+                @Localizer["Configure "]
+                <MudIcon Icon="@Icons.Material.Filled.Settings"/>
```

Comment:
Instead of the ` ` on the localization I think you can achieve the same effect by putting the `ml-2` class on the icon